opm/simulators/wells/MultisegmentWell_impl.hpp

```diff
@@ -3286,9 +3302,47 @@ namespace Opm
 
 
 
+    // TODO: the three ICD functions probably can be refactored a little bit to reduce some code duplication
```

Comment:
I assume that the assembleSICDPressureEq() function is unchanged, and it is just git that thinks it is the new one?

Reply:
Yes. Git got confused with the two functions, the name only has one letter difference.